komga-webui/src/views/BookReader.vue

```diff
@@ -1,284 +1,216 @@
 <template>
-  <div v-if="pages.length > 0" style="background: black; width: 100%; height: 100%">
-    <!--  clickable zone: left  -->
-    <div @click="rtl ? next() : prev()"
-         class="left-quarter full-height"
-         style="z-index: 1; position: absolute"
-    />
-
-    <!--  clickable zone: menu  -->
-    <div @click="showMenu = true"
-         class="center-half full-height"
-         style="z-index: 1; position: absolute"
-    />
-
-    <!--  clickable zone: right  -->
-    <div @click="rtl ? prev() : next()"
-         class="right-quarter full-height"
-         style="z-index: 1; position: absolute"
-    />
+  <v-container class="ma-0 pa-0 full-height" fluid v-if="pages.length > 0" style="width: 100%;"
+       v-touch="{
+       left: () => next(),
+      right: () => prev(),
+      }"
+  >
+    <div>
+    <v-toolbar
+      dense elevation="1"
+      :class="`settings ${toolbar ? '' : 'd-none'}`"
```

Comment:
any reason to use this instead of `v-if="toolbar"` ? `v-if` works with transitions, while `d-none` doesn't.

Reply:
toolbars don't have a model for hiding

Reply:
entirely forgot about v-if